src/utilities/device_utils.h

```diff
@@ -227,13 +227,15 @@ struct hypre_DeviceData
    rocsparse_handle                  cusparse_handle;
 #endif
 
-#if defined(HYPRE_USING_CUDA) || defined(HYPRE_USING_DEVICE_OPENMP)
+#if defined(HYPRE_USING_CUDA_STREAMS)
```

Comment:
Adding this macro breaks the sycl build for me. There is a related change in `hypre_DeviceDataStream()` in `device_utils.c` where you've also moved the construction of the sycl queue behind this `HYPRE_USING_CUDA_STREAMS` macro. What does is this macro supposed to turn on/off?

Reply:
`HYPRE_USING_CUDA_STREAMS` is whether hypre should create a new stream to run or use the default stream.